### PR TITLE
renew example query results

### PR DESCRIPTION
**Pull request by ofek**
cc @hugovk @tswast

### CODE REVIEW

**Review by tswast — APPROVED** [review]
Thanks!

File: README.rst [hunk 1]
```diff
@@ -251,47 +232,38 @@ Most popular projects in the past year
 
     $ pypinfo --days 365 "" project
     Served from cache: False
-    Data processed: 87.84 GiB
-    Data billed: 87.84 GiB
-    Estimated cost: $0.43
+    Data processed: 1.69 TiB
+    Data billed: 1.69 TiB
+    Estimated cost: $8.45
```
Comment (tswast) [inline README.rst:237]: Ouch! Good we warn people for this one.

Comment (ofek) [reply README.rst:237]: Yes this is a bummer

**Review by hugovk — APPROVED** [review]

**Comment by hugovk** [comment]
Thanks!

**Pull request merged**